luxonis_ml/tracker/tracker.py

```diff
@@ -500,6 +500,30 @@ def upload_artifact(
                 )  # Stores details for retrying later
                 self.log_stored_logs_to_mlflow()
 
+    @rank_zero_only
+    def log_matrix(
+        self, matrix: np.ndarray, name: str = "confusion_matrix"
```

Comment:
Let's also add step as a function parameter and when logging let's include step in the json name e.g. `{name}_{step}.json`